api/server/sdk/cloud_backup.go

```diff
@@ -44,15 +44,19 @@ func (s *CloudBackupServer) Create(
 	}
 
 	// Create the backup
-	if err := s.driver.CloudBackupCreate(&api.CloudBackupCreateRequest{
+	r, err := s.driver.CloudBackupCreate(&api.CloudBackupCreateRequest{
 		VolumeID:       req.GetVolumeId(),
 		CredentialUUID: req.GetCredentialId(),
 		Full:           req.GetFull(),
-	}); err != nil {
+		TaskID:         req.GetTaskId(),
+	})
+	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Failed to create backup: %v", err)
 	}
 
-	return &api.SdkCloudBackupCreateResponse{}, nil
+	return &api.SdkCloudBackupCreateResponse{
+		TaskId: r.TaskID,
```

Comment:
This is really confusing. there is a task id from the caller and a task id from CloudBackupCreate. I do not think the caller should provide an id. Is it necessary for a use case?

Reply:
Hope above comments are helpful and no more confusion. Again, callers do not have to provide it.  They can, if they want this call to be idempotent.

Reply:
I think we should force them to provide it. We do this in Volume Create and in Snapshot Create. We should force them here also. Also in both of those it is called `name`. Suggesting that we call it `name` of the backup here also to be consistent with other creations.